### ISSUE TITLE
Event ids are not matching 

### ISSUE THREAD

**Issue by BastienLacave** [SST-1M-collaboration/sst1mpipe#23]
In the DL1 stereo files (maybe also for mono) of the daily analysis, the event ids in
/dl1/event/telescope/parameters/tel_xxx
and
dl1/event/subarray/trigger
are not matching. 

In /dl1/event/telescope/parameters/tel_xxx, all event ids are unique, and I believe they are correct.
In dl1/event/subarray/trigger, less than 12% are found in /dl1/event/telescope/parameters/tel_xxx, and there are duplicates of the same event id (even triplets), I believe this is not expected.

I was looking into this as CTLearn uses the event ids from the trigger, preventing me from matching theme with the DL2 events of the RF method.

**Comment by BastienLacave** [comment]
this one is from 29/01/2024.
I am not sure about the version of the pipeline however

**Comment by jurysek** [comment]
Yes, this is very dependent on the data file you are processing and on the pipeline version. From 20240130 SWAT tags coincident events and arrayEvtNum is used as event_id. This particular event_id is used for stereo event matching in sst1mpipe >=0.3.3. Before that date and version, I cannot guarantee what event_id is in the /trigger, because we did not care too much.

I have just checked on some random data file from 20240213 using sst1mpipe v0.3.3, and I can confirm that event ids match in both tables (parameters and trigger) for both mono and stereo files.

But if this is still an issue in 0.4.1 run on data older than 20240130, please let us know and we can debug it.
